tests/integration/records/views/test_views_conferences.py

```diff
@@ -62,5 +62,13 @@ def test_conference_record_search_results(api_client, db, es_clear, create_recor
     expected_metadata = record.serialize_for_es()
 
     result = api_client.get("/conferences")
+    result_created = result.json["hits"]["hits"][0]["created"]
```

Comment:
These tests are dups of ```serializers/json`` we just need to make sure we are returning the proper status codes here WDYT?

Reply:
yeah I agree we can remove the assertions including `metadata`. I think some of the serializer tests don't have `created`, `updated` assertion but we can add them there.